infinispan-client-quickstart/src/main/resources/application.properties

```diff
@@ -1,3 +1,5 @@
 # Your configuration properties
 quarkus.http.port=8081
-quarkus.infinispan-client.server-list=localhost:11222
\ No newline at end of file
+quarkus.infinispan-client.server-list=localhost:11222
+## Docker 4 Mac workaround
+quarkus.infinispan-client.client-intelligence=BASIC
```

Comment:
How does it authenticate now that we have a username and password?

Reply:
I don't know why it was working just before without any auth configured. but I'm getting very weird stuff in my laptop.
Would you mind testing the quickstart and the docker-compose file from your side, please?

Reply:
on it.

Reply:
works!